src/doc/intro.md

```diff
@@ -392,20 +392,21 @@ Here's an example of a concurrent Rust program:
 use std::thread::Thread;
 
 fn main() {
-    for _ in range(0u, 10u) {
-        Thread::spawn(move || {
+    let guards: Vec<_> = (0..10).map(|_| {
+        Thread::scoped(|| {
             println!("Hello, world!");
-        });
-    }
+        })
+    }).collect();
```

Comment:
Well, if you don't `collect`, the `map` won't do anything, as it's lazy. If you switch to a `for`, each guard drops out of scope on each iteration, so they end up executing serially anyway.